cli/cmd/user/list.go

```diff
@@ -23,42 +27,54 @@ func ListCmd(cfg *config.Config) *cobra.Command {
 			defer client.Close()
 
 			if projectName != "" {
-				members, err := client.ListProjectMembers(context.Background(), &adminv1.ListProjectMembersRequest{
-					Organization: cfg.Org,
-					Project:      projectName,
-				})
-				if err != nil {
-					return err
-				}
+				if strings.HasPrefix(pageToken, "usr") {
+					err = listProjectMembers(cmd, client, cfg.Org, projectName, strings.TrimPrefix(pageToken, "usr"), pageSize)
+					if err != nil {
+						return err
+					}
+				} else if strings.HasPrefix(pageToken, "inv") {
+					err = listProjectInvites(cmd, client, cfg.Org, projectName, strings.TrimPrefix(pageToken, "inv"), pageSize)
+					if err != nil {
+						return err
+					}
+				} else {
+					err = listProjectMembers(cmd, client, cfg.Org, projectName, pageToken, pageSize)
+					if err != nil {
+						return err
+					}
 
-				invites, err := client.ListProjectInvites(context.Background(), &adminv1.ListProjectInvitesRequest{
-					Organization: cfg.Org,
-					Project:      projectName,
-				})
-				if err != nil {
-					return err
+					cmd.Println()
```

Comment:
This will cause a dangling newline in cases where there are no invites or no members